### PR TITLE
#7172 Move logging for false properties from settings to config

### LINKED ISSUE #7172: False Logging Warnings from new Properties
Following our recent improvements to the metrics logging, there is still a persisting issue with some warnings that continue to be logged. This seems to be due to configurations that were added only to the new framework and were never integrated into Settings in the first place.

To completely eliminate these unnecessary warnings, we need to take the following steps:

```[tasklist]
### Tasks
- [ ] Remove Warning Logic from Settings.java
- [ ] Incorporate Old Settings into the New Framework
- [ ] Add Warning Logic to the New Configuration Engine
```

### PR DESCRIPTION
**Pull request by timo0**
Fix #7172

### CODE REVIEW

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/internal/PlatformConfigUtils.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+public class PlatformConfigUtils {
+    private static final Logger logger = LogManager.getLogger(PlatformConfigUtils.class);
+
+    private PlatformConfigUtils() {
+        // Utility class
+    }
+
+    public static void logNotKnownConfigProperties(@NonNull final Configuration configuration) {
+        Objects.requireNonNull(configuration);
```
Comment (hendrikebbers) [inline platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/internal/PlatformConfigUtils.java:40]:
```suggestion
        Objects.requireNonNull(configuration, "configuration must not be null");
```

**Review by hendrikebbers — DISMISSED** [review]

**Review by cody-littley — APPROVED** [review]
A bit of a hack, but Timo explained the reasoning to me. I think this is worth doing sooner rather than later, even if the solution we use now is not the solution we will use eventually.

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/internal/PlatformConfigUtils.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import com.swirlds.config.api.Configuration;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class PlatformConfigUtils {
```
Comment (hendrikebbers) [inline platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/internal/PlatformConfigUtils.java:32]: Javadoc

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/internal/PlatformConfigUtils.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    private static final Logger logger = LogManager.getLogger(PlatformConfigUtils.class);
+
+    private PlatformConfigUtils() {
+        // Utility class
+    }
+
+    public static void logNotKnownConfigProperties(@NonNull final Configuration configuration) {
```
Comment (hendrikebbers) [inline platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/internal/PlatformConfigUtils.java:39]: javadoc

**Review by hendrikebbers — APPROVED** [review]

**Pull request merged**